python/pyarrow/tests/test_jvm.py

```diff
@@ -68,6 +69,20 @@ def _jvm_field(jvm_spec):
     return om.readValue(jvm_spec, pojo_Field)
 
 
+def _jvm_schema(jvm_spec, metadata=None):
+    field = _jvm_field(jvm_spec)
+    schema_cls = jpype.JClass('org.apache.arrow.vector.types.pojo.Schema')
+    fields = jpype.JClass('java.util.ArrayList')()
+    fields.add(field)
+    if metadata:
+        dct = jpype.JClass('java.util.HashMap')()
+        for k, v in six.iteritems(metadata):
```

Comment:
`metadata.items()` would have been enough, `iteritems` is only useful if you have a very large mapping.